bundle/config/resources_test.go

```diff
@@ -118,57 +117,55 @@ func TestResourcesBindSupport(t *testing.T) {
 	supportedResources := &Resources{
 		Jobs: map[string]*resources.Job{
 			"my_job": {
-				JobSettings: &jobs.JobSettings{},
+				JobSettings: jobs.JobSettings{},
 			},
 		},
 		Pipelines: map[string]*resources.Pipeline{
 			"my_pipeline": {
-				CreatePipeline: &pipelines.CreatePipeline{},
+				CreatePipeline: pipelines.CreatePipeline{},
 			},
 		},
 		Experiments: map[string]*resources.MlflowExperiment{
 			"my_experiment": {
-				Experiment: &ml.Experiment{},
+				Experiment: ml.Experiment{},
 			},
 		},
 		RegisteredModels: map[string]*resources.RegisteredModel{
 			"my_registered_model": {
-				CreateRegisteredModelRequest: &catalog.CreateRegisteredModelRequest{},
+				CreateRegisteredModelRequest: catalog.CreateRegisteredModelRequest{},
 			},
 		},
 		Schemas: map[string]*resources.Schema{
 			"my_schema": {
-				CreateSchema: &catalog.CreateSchema{},
+				CreateSchema: catalog.CreateSchema{},
 			},
 		},
 		Clusters: map[string]*resources.Cluster{
-			"my_cluster": {
-				ClusterSpec: &compute.ClusterSpec{},
-			},
+			"my_cluster": {},
 		},
 		Dashboards: map[string]*resources.Dashboard{
 			"my_dashboard": {
-				Dashboard: &dashboards.Dashboard{},
+				Dashboard: dashboards.Dashboard{},
 			},
 		},
 		Volumes: map[string]*resources.Volume{
 			"my_volume": {
-				CreateVolumeRequestContent: &catalog.CreateVolumeRequestContent{},
+				CreateVolumeRequestContent: catalog.CreateVolumeRequestContent{},
 			},
 		},
 		Apps: map[string]*resources.App{
 			"my_app": {
-				App: &apps.App{},
+				App: apps.App{},
 			},
 		},
 		QualityMonitors: map[string]*resources.QualityMonitor{
 			"my_quality_monitor": {
-				CreateMonitor: &catalog.CreateMonitor{},
+				CreateMonitor: catalog.CreateMonitor{},
 			},
 		},
 		ModelServingEndpoints: map[string]*resources.ModelServingEndpoint{
 			"my_model_serving_endpoint": {
-				CreateServingEndpoint: &serving.CreateServingEndpoint{},
+				CreateServingEndpoint: serving.CreateServingEndpoint{},
```

Comment:
FWIW, with these structs being zero-initialized, the assignment can be removed entirely.

Reply:
I agree. For context, most replacement were made with search-and-replace, so they look odd (AI tools were going back and forth on whether it should be a pointer or a struct, so I stopped using those for this).

I feel like there should be a linter to clean this up, but I could not find one.